WordPress/Classes/Jetpack/Utility/SharedDataIssueSolver.swift

```diff
@@ -41,7 +41,7 @@ public final class SharedDataIssueSolver: NSObject {
     /// both apps are logged in with the same account.
     ///
     public func migrateAuthKey(for username: String) {
-        guard AppConfiguration.isJetpack,
+        guard BuildSettings.current.brand == .jetpack,
```

Comment:
Using `BuildSettings` over `AppConfiguration` allows moving to WordPressData where the latter is not available.